Readme.md

```diff
@@ -1,8 +1,28 @@
-install
+Install
 -------
+To install a running SurfCaptain Instance you should use the distribution Package that holds all the
+composer dependencies. Follow theses steps to enter the Setup:
 
 	git clone git@github.com:lightwerk/SurfCaptainDistribution.git
 	cd surfcaptaindistribution
 	composer install
+	copy the content of Packages/Application/Lightwerk.SurfCaptain/Configuration/Routes.yaml to Configuration/Routes.yaml
```

Comment:
I'll do it later but I want the rest of the docu improvement to be available since I get asked that alot.